code/modules/mob/living/basic/space_fauna/spider/spiderlings/spiderling_subtypes.dm

```diff
@@ -3,23 +3,82 @@
 /// Will differentiate into the base giant spider (known colloquially as the "guard" spider).
 /mob/living/basic/spiderling/guard
 	grow_as = /mob/living/basic/giant_spider
+	name = "guard spiderling"
+	desc = "Furry and brown, it makes you shudder to look at it. This one has sparkling red eyes."
+	icon_state = "guard_spiderling"
+	icon_dead = "guard_spiderling_dead"
+	desc = "Furry and brown, specializing in defense of the hive and other spides."
```

Comment:
This is probably supposed to be `menu_description`

Reply:
yeah it is